packages/components/Button/src/Button.tsx

```diff
@@ -20,12 +26,16 @@ export const Button = compose<IButtonType>({
       content,
       onAccessibilityTap = userProps.onClick,
       accessibilityLabel = userProps.content,
+      componentRef = userProps.componentRef != undefined ? userProps.componentRef : React.useRef(null),
```

Comment:
I think just `componentRef = React.useRef(null)` should do what you want!

Reply:
What if the user provides their own ref?

Reply:
I'm pretty sure the assignment operator here is for providing default values during the pattern matching - so if `userProps.componentRef` is defined it won't assign it to `React.useRef(null)`

Reply:
I see, that seems to be the case. Thanks for the explanation! I'll update it in my next iteration.